CI/macos/02_build_obs.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+##############################################################################
+# macOS build function
+##############################################################################
+#
+# This script file can be included in build scripts for macOS or run directly
+#
+##############################################################################
+
+# Halt on errors
+set -eE
+
+build_obs() {
+    status "Build OBS"
+    if [ -z "${CI}" ]; then
+        _backup_artifacts
+    fi
+    step "Configure OBS..."
+    _configure_obs
+
+    ensure_dir "${CHECKOUT_DIR}/"
+    step "Build OBS targets..."
+    cmake --build ${BUILD_DIR}
+}
+
+bundle_obs() {
+    status "Creation of relocatable macOS application bundle"
```

Comment:
Starting with a verb is probably more clear, here and elsewhere.  Either "Create" or "Creating" is fine unless there is more context to this message where "Creation" makes more sense.